File: src/main/java/com/github/peanutbutter/unicorn/tarvester/model/products/InStoreOnly.java
package com.github.peanutbutter.unicorn.tarvester.model.products;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record InStoreOnly(
        @JsonProperty("availability_status")
        String availabilityStatus) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/InStoreOnly.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class InStoreOnly(
    @param:JsonProperty("availability_status") val availabilityStatus: String
) {}
